src/DateRangePicker.jsx

```diff
@@ -496,12 +498,13 @@ const DateRangePicker = React.createClass({
   },
 
   render: function() {
-    let {paginationArrowComponent: PaginationArrowComponent, numberOfCalendars, stateDefinitions, selectedLabel, showLegend, helpMessage} = this.props;
+    let {paginationArrowComponent: PaginationArrowComponent, className, numberOfCalendars, stateDefinitions, selectedLabel, showLegend, helpMessage} = this.props;
 
     let calendars = Immutable.Range(0, numberOfCalendars).map(this.renderCalendar);
+    className = this.cx({element: null}) + ' ' + className;
 
     return (
-      <div className={this.cx({element: null})}>
+      <div className={className.trim()}>
```

Comment:
There will be a trailing space if no `className` is given, but this is not really a problem.
Another idea... What are your thoughts on this? 😉
```
let combinedClassName = [ this.cx({element: null}), className ].join(' ').trim()
<div className={combinedClassName}>
```